### PR TITLE
Feature/create user update

### PR DESCRIPTION
**Pull request by Nicolas-Rodriguez-Ch**
## Summary
- Proposal update & create user service. 
- Records can be created in the user phone number and addresses dependencies.

### CODE REVIEW

File: src/api/users/users.controllers.ts [hunk 1]
```diff
@@ -45,7 +45,8 @@ import {
   res: Response,
  ) => {
   try {
-    const user = await createUser(req.body);
+    console.log(req.body);
```
Comment (ht1204) [inline src/api/users/users.controllers.ts:48]: what is this console.log doing?

File: src/api/users/users.services.ts [hunk 2]
```diff
@@ -34,7 +34,22 @@ export const getUserById = (id: string) => {
 }
 
 export const createUser = (input: any) => {
-  const { email, password, name, last_name, document_type, document_number, date_of_birth, city, contact_email, contact_sms, contact_wpp, user_role } = input;
+  const { 
+    email,
```
Comment (ht1204) [inline src/api/users/users.services.ts:38]: What is this blank space doing?

Comment (Nicolas-Rodriguez-Ch) [reply src/api/users/users.services.ts:38]: forgot to erase it, resolved.

File: src/api/users/users.controllers.ts (hunk 1 above)
Comment (Nicolas-Rodriguez-Ch) [reply src/api/users/users.controllers.ts:48]: I forgot to delete it 😅, resolved.

**Comment by Nicolas-Rodriguez-Ch** [comment]
new commit with changes according to feedback

File: src/api/users/users.controllers.ts (hunk 1 above)
Comment (ht1204) [reply src/api/users/users.controllers.ts]: Please, attentive to details, always.

**Pull request merged**